elasticsearch/tpl/assisted-service-events.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
This file is just for local development for now, but in the future we should have some mechanism to make sure elasticsearch objects are created, and use it for both local and integration/stage/prod